Multisig/UI/App/WhatsNewHandler.swift

```diff
@@ -0,0 +1,67 @@
+//
+// Created by Dirk Jäckel on 22.12.21.
+// Copyright (c) 2021 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+import WhatsNewKit
+
+class WhatsNewHandler {
+    private let whatsNew = WhatsNew(
+            // Show the WhatsNew screen only once for users of this version
+            version: WhatsNew.Version(major: 3, minor: 6, patch: 0),
+            title: "What's new",
+            items: [
+                WhatsNew.Item(
+                        title: "Intercom chat support",
+                        subtitle: "Have a trouble or want to leave feedback? Drop us a message directly in the Intercom chat.",
+                        image: UIImage(named: "ico-whats-new-chat")
+                ),
+                WhatsNew.Item(
+                        title: "Initiate asset transfer",
+                        subtitle: "Initiate a transfer of your tokens on-the-go...",
+                        image: UIImage(named: "ico-whats-new-ether")
+                ),
+                WhatsNew.Item(
+                        title: "Execute transactions",
+                        subtitle: "...and execute those transactions from your mobile.",
+                        image: UIImage(named: "ico-whats-new-transactions")
+                )
+            ]
+    )
+
+    var whatsNewViewController: WhatsNewViewController?
+
+    init() {
+        let whatsNews = [whatsNew]
+        var configuration = WhatsNewViewController.Configuration()
+
+        configuration.backgroundColor = .white
```

Comment:
We already have predefined styles for labels & buttons could you reuse them ? 
or define the missing one instead of setting values here?

Reply:
I assume you mean the colors in: UIColor+Styles? 

Unfortunately @TanyaEfremova did not use those names. So I will try to find out why colors look similar to those she used. But it would be really helpful if the designs used the color names from the ios Project.

Reply:
For example: is `tertiaryBackground` or `quaternaryBackground` the right color for the background?

Reply:
See f02d0abba1bdde87d8f01040c8e9a1d854aac4fc for my attempt to use the right colors.